libs/coin-modules/coin-tezos/src/api/index.ts

```diff
@@ -51,18 +52,22 @@ async function craft(
   return rawEncode(contents);
 }
 
-async function estimate(addr: string, amount: bigint): Promise<bigint> {
-  const accountInfo = await api.getAccountByAddress(addr);
+export async function estimate(transactionIntent: TransactionIntent): Promise<bigint> {
```

Comment:
Why do you need to export this function?

Reply:
my bad